### DISCUSSION TITLE
were are stored the notes ?

### DISCUSSION THREAD

**Discussion by moolight-seashell** [TriliumNext/Trilium#4304] [Q&A] [answered]
I'm not sure about how this is working but is the notes are stored somwhere ?

i have a file sync between my devices so can i do something with it or it's only like server based ?

**Comment by zerebos** [comment, marked as answer]
The notes are stored in an sqlite database, not via flat files so it's really not recommended to use a sync service like you're describing. See here: https://github.com/zadam/trilium/wiki/FAQ#can-i-use-dropbox--google-drive--onedrive-to-sync-data-across-multiple-computers

But the directory can be found here: https://github.com/zadam/trilium/wiki/Data-directory#location

**Comment by moolight-seashell** [comment]
yeah ok . that's what i thought i understood and it's a shame not to be able to save everything in a folder or file. like word and those disguised zips or like cherrytree with xml.

however, on cherry tree you can save sql lite in flat files.

well, I'm a bit disappointed because this software is incredible but I couldn't use it.

anyway thanks

**Reply by zerebos** [reply]
The FAQ page (https://github.com/zadam/trilium/wiki/FAQ#why-database-instead-of-flat-files) also has info on why sqlite was chosen over flat files. Generally the underlying thing should not matter to the end use much because they can import/export as needed

**Reply by moolight-seashell** [reply]
i 'dont really understand wy is not recomended to sync a mysql database it look like other app like appflowy store data in a database and there is no porbleme due to sync

can we change the default path were the databse is stored ?

**Reply by meichthys** [reply]
You'd probably be fine but it's always recommended to not sync databases that way. 
Here's some more information: https://www.sqlite.org/useovernet.html

Summary: 

>  Choose the technology that is right for you and your customers. If your data lives on a different machine from your application, then you should consider a client/server database. SQLite is designed for situations where the data and application coexist on the same machine. SQLite can still be made to work in many remote database situations, but a client/server solution will usually work better in that scenario.

**Comment by sottey** [comment]
Additionally, you can choose the root note, right click and choose Export and you will get a zip file with all of the notes in HTML/Markdown form. If this is something you only need to do once in a while, you could automate that.

**Reply by moolight-seashell** [reply]
for me it's basicaly once or mor each day so not very usable

**Reply by meichthys** [reply]
Exporting has been added to the ETAPI so you could script a scheduled export of all your notes if needed: https://github.com/zadam/trilium/issues/3012

**Reply by moolight-seashell** [reply]
the problem is that i cannot  define a suitable schedule
I need the database to be permanently identical, or at least within a very short timeframe.
then I access my notes rather randomly over time

**Reply by meichthys** [reply]
Ok, in that case you'd be better off with a flat-file note taking system. Nextcloud Notes is quite nice with mobile app, web app, etc
